### ISSUE TITLE
Flow chart is not shown correctly

### ISSUE THREAD

**Issue by Dentosal** [hackmdio/hackmd#84]
Some flowcharts are not shown correctly.

For example:

    ```flow
    s=>start: Start
    e=>end: End
    c=>condition: condition
    o=>operation: operation

    s->o
    o->c
    c(yes)->o
    c(no)->e
    ```

**Comment by jackycute** [comment]
it's a flowchart.js issue, but I will try to fix it.
https://github.com/adrai/flowchart.js/issues/67

**Comment by jackycute** [comment]
Fixed this issue and pull request to flowchart.js.
PR had been accpeted and flowchart.js relaesed new version (1.6.1) which contain this fix.

Closed by commit 3e48d46514897f480909c1d775eb04fafd3e7325
